### DISCUSSION TITLE
Refresh Schedule not working

### DISCUSSION THREAD

**Discussion by vetriselvan-98** [getredash/redash#6753] [Q&A]
We have a few redash queries with a refresh schedule set. These queries are powering some of our GSheets using IMPORTDATA but lately we noticed that the sheet isn't getting automatically refreshed nor are the queries refreshing as per schedule. From the image - you can see that the query is set to refresh everyday but it also says "Refreshed 6 days ago". 
![image](https://github.com/getredash/redash/assets/142983224/efc1dfac-7ea3-45c4-b267-bbf6278b1b9a)

**Comment by gigamesh** [comment]
+1

It's also unclear in the documentation how to manually trigger an execution. I get "Internal Server Error" back, which is a really unhelpful message.

**Comment by proxeter** [comment]
Have you correctly set up workers for scheduled queries?
You can watch it on page 'admin/queries/jobs#workers'

![image](https://github.com/getredash/redash/assets/1473996/5b1b6a6e-73f7-4d54-9d12-17f482ece37f)

**Comment by eradman** [comment]
@vetriselvan-98 do these queries have parameters? There was a long-standing bug with this that was fixed recently

https://github.com/getredash/redash/pull/6683
